corehq/apps/locations/views.py

```diff
@@ -407,12 +412,40 @@ def products_form(self):
 
         form = MultipleSelectionForm(
             initial={'selected_ids': self.products_at_location},
-            submit_label=_("Update Product List")
+            submit_label=_("Update Product List"),
+            prefix="products",
         )
         form.fields['selected_ids'].choices = self.all_products
         return form
 
     @property
+    @memoized
+    def all_users(self):
```

Comment:
this isn't actually using elasticsearch for anything other than the domain/mobile filtering and then paring down the fields, right? 

i generally shy away from using it for stuff unless I have to since it creates many support issues whenever the pillows get behind.